.github/workflows/publish-containers.yml

```diff
@@ -132,11 +132,17 @@ jobs:
             Digest: ${{ steps.docker_build.outputs.digest }}
             Tags: ${{ env.CONTAINER_TAGS }}"
 
-      - name: Trigger staging deployment for tagged release
+      - name: Trigger staging deployments for tagged release
         if: ${{ startsWith(github.ref, 'refs/tags/') }}
         run: |
           curl -f -X POST \
             -H "Accept: application/vnd.github.v3+json" \
             -H "Authorization: Bearer ${{ secrets.SALEOR_RELEASE_TOKEN }}" \
             https://api.github.com/repos/saleor/saleor-multitenant/dispatches \
             -d "{\"event_type\":\"deploy-staging\",\"client_payload\":{\"version\":\"$VERSION\"}}"
+
+          curl -f -X POST \
```

Comment:
Wouldn't that trigger for 3.2, 3.1 and 3.3?

Reply:
There's a filter in deployments repo